specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/examples/CosmosDBRestorableMongodbCollectionGet.json

```diff
@@ -0,0 +1,27 @@
+{
+  "parameters": {
+    "api-version": "2020-06-01-preview",
+    "subscriptionId": "subid",
+    "location": "WestUS",
+    "instanceId": "d9b26648-2f53-4541-b3d8-3044f4f9810d",
+    "restorableMongodbCollectionId": "79609a98-3394-41f8-911f-cfab0c075c86"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/subid/providers/Microsoft.DocumentDb/locations/westus/restorableDatabaseAccounts/98a570f2-63db-4117-91f0-366327b7b353/restorableMongodbCollections/79609a98-3394-41f8-911f-cfab0c075c86",
+        "type": "Microsoft.DocumentDB/locations/restorableDatabaseAccounts/restorableMongodbCollections",
+        "name": "79609a98-3394-41f8-911f-cfab0c075c86",
+        "properties": {
+          "resource": {
+            "_rid": "zAyAPQAAAA==",
+            "eventTimestamp": "2020-10-13T04:56:42Z",
+            "ownerId": "Collection1",
+            "ownerResourceId": "V18LoLrv-qA=",
+            "operationType": "Create"
```

Comment:
Not returning the MongoDBCollection in here ??

Reply:
Due to the different ways of getting the Collection content in Mongo 3.2 vs 3.6, we have decided to not include the content at this time.  The crucial infos (ie. operationtype, ownerId, and ownerResourceId) are still available.